CONTRIBUTING.md

```diff
@@ -177,18 +181,31 @@ a new application. The name of the application should consist of the
 component name in lowercase, with words separated by dashes (`-`). The
 deployment server will provide a remote repository URL that will
 contain the app. Add it to the list of remotes by running `git
-remote add gallery-test [deployment server git URL]`.
+remote add [app name]-test [deployment server git URL]`.
 
-#### Step 2: Edit and commit the `Procfile`
-Edit the `Procfile` in the repository to say `gunicorn
-app_name:server`, where `app_name` is the name of the app you want to
-deploy in the `tests/dashbio_demos/` folder. Then, commit the change
-you made to the `Procfile`.
+#### Step 2: Edit and commit app-specific files
+
+##### Step 2a: Edit the `Procfile`
+Edit the `Procfile` at the root of the repository to say `gunicorn
+tests.dashbio_demos.app_name:server`, where `app_name` is the name of
+the app you want to deploy in the `tests/dashbio_demos/` folder. 
+
+##### Step 2b: Edit `config.py`
+Edit the `config.py` file at the root of the repository such that the variable
+`DASH_APP_NAME` be set to the name of your app, but with `app`
+replaced by `dash` and underscores (`_`) replaced by dashes
+(`-`). (e.g., for `app_manhattan_plot`, the `DASH_APP_NAME` variable
+should be set to `dash-manhattan-plot`.)
+
+##### Step 2c: Commit the changes
+Commit the `Procfile` and `config.py`, but *do not push to the
+`dash-bio` repo*!
 
 #### Step 3: Push to the playground server
-Run `git push gallery-test master`. This will deploy the app on the
-playground server. Test that it works by visiting the URL that is
-displayed in the console.
+Run `git push [app name]-test master`. This will deploy the app
+on the playground server. Test that it works by visiting the URL that
```

Comment:
I would write "Check" instead of "Test" to distinguish from actual Integration tests

Reply:
@Bachibouzouk addressed in 8c4ecd4!